frappe/utils/jinja_globals.py

```diff
@@ -75,7 +76,9 @@ def web_blocks(blocks):
 		# see render_dynamic method web_page.py
 		if template not in frappe.flags.web_block_scripts:
 			for script in scripts:
-				html += f"<script data-web-template='{template}'>{script}</script>"
+				html += re.sub(
+					"<script", f"<script data-web-template='{template}'", script, count=1, flags=re.I
+				)
```

Comment:
`include_script` is more suitable for this unless I missed something here. 

usage:

https://github.com/frappe/frappe/blob/9d97097c12e6325a8ce7cdbe8a59e1193ceb75e4/frappe/www/app.html#L50-L52